data/card_catalogue/anger.json

```diff
@@ -71,11 +83,11 @@
     "tier": "green",
     "details": {
       "rules": {
-	"active": "Summon one of the following items: a knife, 5 feet of rope, a match, or a bottle of water",
-	"passive": "Get triple advantage on all rolls when below 5 health"
+        "active": "Summon one of the following items: a knife, 5 feet of rope, a match, or a bottle of water.",
+        "passive": "Triple advantage on all d20 rolls when below 5 health."
```

Comment:
Looking at this now I think it's okay for it to apply to damage too, 5 health isn't much.

Reply:
Sounds good to me. I think it might be broken in that damage in the game is primarily dealt through being forced to use a card rather than actually being hit, but it's broken in such a way that abusing that would be fun

Reply:
One of my long term visions for cardmaster is a system of banning cards that works as follows:
- Card bannings are announced 1 session before they go into effect
- You to pick another card of the same rarity of your choice to replace the banned cards in your hand
- We produce a line of effects that let you cast or otherwise use banned cards temporarily

Reply:
Another card of the same rarity might not be good enough, it might require another reward. But ultimately it should feel rewarding to have identified and hoarded unfair cards.

Reply:
- #120